addons/website_event/static/src/snippets/options.js

```diff
@@ -62,7 +62,7 @@ options.registry.WebsiteEvent = options.Class.extend({
      * @private
      */
     _getEventObjectId() {
-        const objectIds = this.currentWebsiteUrl.match(/-\d+(?![-\w\d])/);
-        return parseInt(objectIds[0].replace('-', '')) | 0;
+        const objectIds = this.currentWebsiteUrl.match(/\d+(?![-\w\d])/);
```

Comment:
the original expression looked for the first number that is:

* after a -
* not directly followed by a
    * `-`
    * `\w`: alphanumeric character and _
    * `\d`: numeric character but they already included in `\w` and doesn't make sense anyway since numeric character are already consumed by `\d+`

So with your change it still seems good, but you could probably remove \d while you are at it to make the expression simpler.

```suggestion
        const objectIds = this.currentWebsiteUrl.match(/\d+(?![-\w])/);
```